src/Tvheadend.cpp

```diff
@@ -1060,6 +1060,22 @@ PVR_ERROR CTvheadend::DeleteTimer
     /* EPG-query-based repeating timer */
     return m_autoRecordings.SendAutorecDelete(timer);
   }
+  else if ((timer.iTimerType == TIMER_ONCE_CREATED_BY_TIMEREC) ||
+           (timer.iTimerType == TIMER_ONCE_CREATED_BY_AUTOREC))
+  {
```

Comment:
As long as timers created by autorecs and timerecs are read-only, UpdateTimer works correct, yes. Abort recording is "different" and therefore okay even if the timer is read-only.

And I made timers created by autorecs and timerecs read-only for a reason. At least with tvh 4.0, although tvh webui allows editing these timers, former or later all changes will be lost, latest after a restart of tvh. At least this was the behavior when I last checked. Don't know whether this works correctly with tvh 4.1 or was fix for recent 4.0.x